.github/workflows/test_and_deploy.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
Note here (and below in other references to Python versions), Gymnasium 0.29.1 is built/tested only up to 3.11. We've pinned to that version of Gymnasium as later releases are alpha. We should probably not add 3.12 until Gymnasium releases support it.

Reply:
Gymnasium is a pure Python lib. And while it is tagged as being available for 3.8-3.11 versions. It only has minimal requirement for the Python version being 3.8+. From my experience, Gymnasium 0.29 already works on Python 3.12 without a problem. But if you want to be on the safe side, sure let's wait for the official support.